ecosystem/ffmpeg_plugin/build_ffmpeg_plugin.sh

```diff
@@ -1,19 +1,51 @@
 #!/bin/bash
 
 # SPDX-License-Identifier: BSD-3-Clause
-# Copyright 2024 Intel Corporation
+# SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
```

Comment:
Is this change intentional?

Reply:
Yes it's recommended to use this format as this is still 2024 copyright i decided to change it